content/blog/announcing-enum-support/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Announcing Cross-Language Enum Support"
+date: 2020-12-14
```

Comment:
Might want to update this to be today.

Reply:
It is today. I'm guessing you mean whichever today that this actually goes out on?

Reply:
Yeah - sorry - my bad - I was reviewing based on an earlier preview link.